### ISSUE TITLE
Images with historical compiler versions

### ISSUE THREAD

**Issue by 0xEAB** [dlang-dockerized/packaging#2]
[LDC 1.34](https://github.com/ldc-developers/ldc/releases/tag/v1.34.0) was released a few days ago.

**Comment by 0xEAB** [comment]
Versions older than a few years are prolly quite a hassle to setup on up-to-date systems:
[legacy-dmd@dlang-dockerized/packaging](https://github.com/dlang-dockerized/packaging/tree/legacy-dmd)

LDC is a pita, too, as historical LLVM versions don’t build out-of-the-box on modern system either.
(LDC v0.17 that was ported to LLVM v7 is a positive exception.)

**Comment by 0xEAB** [comment]
Implemented for LDC 8443078b1bf7818e6d82739bf57ea4ba6b5c5a45.

**Comment by 0xEAB** [comment]
### DUB versions for DMD
```
2.109.1
DUB version 1.38.1, built on Jun 30 2024
2.109.0
DUB version 1.38.0, built on Jun  1 2024
2.108.1
DUB version 1.37.0, built on May  2 2024
2.108.0
DUB version 1.37.0, built on Apr  1 2024
2.107.1
DUB version 1.36.0, built on Mar  1 2024
2.107.0
DUB version 1.36.0, built on Feb  1 2024
2.106.1
DUB version 1.35.1, built on Jan  1 2024
2.106.0
DUB version 1.35.0, built on Dec  2 2023
2.105.3
DUB version 1.34.0, built on Nov  1 2023
2.105.2
DUB version 1.34.0, built on Sep 15 2023
2.105.1
DUB version 1.34.0, built on Sep  1 2023
2.105.0
DUB version 1.34.0, built on Aug  1 2023
2.104.2
DUB version 1.33.1, built on Jul 15 2023
2.104.1
DUB version 1.33.1, built on Jul  2 2023
2.104.0
DUB version 1.33.0, built on Jun  2 2023
2.103.1
DUB version 1.32.1, built on May  1 2023
2.103.0
DUB version 1.32.0, built on Apr  1 2023
2.102.2
DUB version 1.31.1, built on Mar  1 2023
2.102.1
DUB version 1.31.1, built on Feb 15 2023
2.102.0
DUB version 1.31.0, built on Feb  1 2023
2.101.2
DUB version 1.30.0, built on Jan  1 2023
2.101.1
DUB version 1.30.0, built on Dec 15 2022
2.101.0
DUB version 1.30.0, built on Nov 14 2022
2.100.2
DUB version 1.29.2, built on Sep 10 2022
2.100.0
DUB version 1.29.0, built on May 13 2022
2.099.1
DUB version 1.28.0, built on Apr  7 2022
2.099.0
DUB version 1.28.0, built on Mar  6 2022
2.098.1
DUB version 1.27.0, built on Dec 19 2021
2.098.0
DUB version 1.27.0, built on Oct 10 2021
2.097.2
DUB version 1.26.1, built on Aug  9 2021
2.097.1
DUB version 1.26.1, built on Jul 20 2021
2.097.0
DUB version 1.26.0, built on Jun  3 2021
2.096.1
DUB version 1.25.0, built on Apr 23 2021
2.096.0
DUB version 1.25.0, built on Mar 11 2021
2.095.1
DUB version 1.24.1, built on Feb 18 2021
2.095.0
DUB version 1.24.0, built on Jan  1 2021
2.094.2
DUB version 1.23.0, built on Nov 20 2020
2.094.1
DUB version 1.23.0, built on Oct 18 2020
2.094.0
DUB version 1.23.0, built on Sep 22 2020
2.093.1
DUB version 1.22.0, built on Aug 15 2020
2.093.0
DUB version 1.22.0, built on Jul  7 2020
2.092.1
DUB version 1.21.0, built on Jun 11 2020
2.092.0
DUB version 1.21.0, built on May 11 2020
2.091.1
DUB version 1.20.1, built on Apr 17 2020
2.091.0
DUB version 1.20.0, built on Mar  9 2020
2.090.1
DUB version 1.19.0, built on Feb  9 2020
2.090.0
DUB version 1.19.0, built on Jan  5 2020
2.089.1
DUB version 1.18.0, built on Dec 14 2019
2.089.0
DUB version 1.18.0, built on Nov  2 2019
2.088.1
DUB version 1.17.0, built on Oct 11 2019
2.088.0
DUB version 1.17.0, built on Sep  1 2019
2.087.1
DUB version 1.16.0, built on Aug  4 2019
2.087.0
DUB version 1.16.0, built on Jul  2 2019
2.086.1
DUB version 1.15.0, built on Jun 15 2019
2.086.0
DUB version 1.15.0, built on May  4 2019
2.085.1
DUB version 1.14.0, built on Apr  5 2019
2.085.0
DUB version 1.14.0, built on Mar  2 2019
2.084.1
DUB version 1.13.0, built on Feb  9 2019
2.084.0
DUB version 1.13.0, built on Jan  2 2019
2.083.1
DUB version 1.12.1, built on Dec  8 2018
2.083.0
DUB version 1.12.0, built on Nov  1 2018
2.082.1
DUB version 1.11.0, built on Oct 11 2018
```
Scraped using <https://github.com/dlang-dockerized/packaging/blob/4503d29e720bc5b35955eb720d5b28aeca7f7551/tools/determine-dub-version-of-dmd-release.sh>.

**Comment by 0xEAB** [comment]
#### Funfact: Sizes of the uncompressed DMD releases (Linux)
```
300M	2.082.1
302M	2.083.0
302M	2.083.1
305M	2.084.0
304M	2.084.1
316M	2.085.0
318M	2.085.1
322M	2.086.0
322M	2.086.1
326M	2.087.0
327M	2.087.1
329M	2.088.0
329M	2.088.1
307M	2.089.0
304M	2.089.1
306M	2.090.0
306M	2.090.1
293M	2.091.0
293M	2.091.1
294M	2.092.0
294M	2.092.1
274M	2.093.0
274M	2.093.1
243M	2.094.0
243M	2.094.1
243M	2.094.2
246M	2.095.0
246M	2.095.1
278M	2.096.0
279M	2.096.1
287M	2.097.0
289M	2.097.1
289M	2.097.2
294M	2.098.0
294M	2.098.1
270M	2.099.0
270M	2.099.1
271M	2.100.0
271M	2.100.2
275M	2.101.0
275M	2.101.1
275M	2.101.2
276M	2.102.0
276M	2.102.1
274M	2.102.2
275M	2.103.0
275M	2.103.1
274M	2.104.0
274M	2.104.1
274M	2.104.2
275M	2.105.0
275M	2.105.1
275M	2.105.2
276M	2.105.3
278M	2.106.0
278M	2.106.1
284M	2.107.0
284M	2.107.1
284M	2.108.0
284M	2.108.1
285M	2.109.0
285M	2.109.1
```

**Comment by 0xEAB** [comment]
Implemented for DMD in f39310dd9c35a1809f27007aa18a31abfcf61c75
